tasks/setup_wp_directories.yml

```diff
@@ -23,33 +35,22 @@
     group: www-data
   when: wordpress_import_template
 
-- name: Remove anonymous user account
-  mysql_user:
-    name: ''
-    host: localhost
-    state: absent
-
-- name: Create a new database with name '{{ wordpress_mysql_database_name }}'
-  mysql_db:
-    name: "{{ wordpress_mysql_database_name }}"
-    state: present
-
-- name: Create mariadb user
-  mysql_user:
-    name: "{{ wordpress_mysql_user }}"
-    password: "{{ wordpress_mysql_password }}"
-    priv: '{{ wordpress_mysql_database_name }}.*:ALL,GRANT'
-    state: present
+- name: Check if wp-config.php already exists
+  stat:
+    path: /var/www/wordpress/wp-config.php
+  register: wp_config
 
 - name: Fetch random salts for WordPress config
   local_action: command curl https://api.wordpress.org/secret-key/1.1/salt/
   register: "wp_salt"
   become: no
+  when: not wp_config.stat.exists
 
 - name: Write wp-config.php
   template:
     src: templates/wp-config.php.j2
     dest: /var/www/wordpress/wp-config.php  
+  when: not wp_config.stat.exists
```

Comment:
Why do you want to create this config only the first time?
It is templated and should be written every time, e.g. if the database connection is changed lateron.